virtualizarr/accessor.py

```diff
@@ -39,19 +40,42 @@ def to_zarr(self, storepath: str) -> None:
         dataset_to_zarr(self.ds, storepath)
 
     def to_icechunk(
-        self, store: "IcechunkStore", append_dim: Optional[str] = None
+        self,
+        store: "IcechunkStore",
+        append_dim: Optional[str] = None,
+        last_updated_at: Optional[datetime] = None,
     ) -> None:
         """
-        Write an xarray dataset to an Icechunk store.
+                Write an xarray dataset to an Icechunk store.
 
-        Any variables backed by ManifestArray objects will be be written as virtual references, any other variables will be loaded into memory before their binary chunk data is written into the store.
+                Any variables backed by ManifestArray objects will be be written as virtual references, any other variables will be loaded into memory before their binary chunk data is written into the store.
 
-        If `append_dim` is provided, the virtual dataset will be appended to the existing IcechunkStore along the `append_dim` dimension.
+                If `append_dim` is provided, the virtual dataset will be appended to the existing IcechunkStore along the `append_dim` dimension.
 
-        Parameters
-        ----------
-        store: IcechunkStore
-        append_dim: str, optional
+                If `last_updated_at` is provided, it will be used as a checksum for any virtual chunks written to the store with this operation.
+                At read time, if any of the virtual chunks have been updated since this provided datetime, an error will be raised.
+                This protects against reading outdated virtual chunks that have been updated since the last read. When not provided, no check is performed.
+                This value is stored in Icechunk with seconds precision, so be sure to take that into account when providing this value.
+
+                Parameters
+                ----------
+                store: IcechunkStore
+                append_dim: str, optional
+                    When provided, specifies the dimension along which to append the virtual dataset.
+                last_updated_at: datetime, optional
+                    When provided, uses provided datetime as a checksum for any virtual chunks written to the store with this operation.
+                    When not provided (default), no check is performed.
+
+        Examples
+        --------
+        To ensure an error is raised if the files containing referenced virtual chunks are modified at any time from now on, pass the current time to ``last_updated_at``.
```

Comment:
This incorrect indenting is probably what's causing the docs failure. The docs fail on any warning, and it's warning with

```
/home/docs/checkouts/readthedocs.org/user_builds/virtualizarr/conda/368/lib/python3.13/site-packages/virtualizarr/accessor.py:docstring of virtualizarr.accessor.VirtualiZarrDatasetAccessor.to_icechunk:13: CRITICAL: Unexpected section title.
```

Reply:
That was it! Thanks!